Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM alpine:latest AS builder
+
+RUN apk update && apk add --no-cache \
+    git \
+    build-base \
+    openssl-dev \
+    libpcap-dev \
+    linux-headers \
+    musl-dev
+
+RUN git clone https://github.com/hufrea/byedpi /opt/byedpi
```

Comment:
There is no need for cloning the repo, because we already have it with Dockerfile. So, `COPY . /opt/byedpi`.

Reply:
> There is no need for cloning the repo, because we already have it with Dockerfile. So, `COPY . /opt/byedpi`.

Sometimes users just downloads one Dockerfile at all, instead of full repo

Reply:
Anyway, fetching specified version always better than fetching latest master

Reply:
> Anyway, fetching specified version always better than fetching latest master

Great advice, thanks! Fixed to latest release code.